Configurations/set-git-version-info.sh

```diff
@@ -22,8 +22,8 @@ if [ -z "$gitversion" ] ; then
     exit 0
 fi
 
-# remove everything before the first "-" to keep the hash part only
-versionsuffix=${gitversion#*-};
+# remove everything before the last "-" to keep the hash part only
+versionsuffix=${gitversion##*-};
```

Comment:
This comment was slightly wrong. We actually used to append both a counter (since the last tag) and a git short hash. For example on `master`, we currently get `8-gf7d830d9`. This counter does strengthen the uniqueness of this identifier a bit more, so we may not want to lose it.

Reply:
In short, everything past the 2nd last dash.

Reply:
But anyway.. we will need to actually feed this script with the marketing version when ran in this workflow. Because in this workflow we actually create the tag *afterwards*.

Reply:
We also need to use MARKETING_VERSION in this script now, not CURRENT_PROJECT_VERSION

Reply:
Ahh also, when the counter is zero (when we're directly at the tag), git describe outputs exactly that tag name, with no hash info (eg `2.0.0-beta.1`). But as this example shows there can still be a dash due to the prerelease naming scheme. When there is a hash, you can be guaranteed it ends in `-g*` though.

Reply:
Or replicate/check release-move-tag.sh logic if you want to detect if we are more than 0 commits away from latest tag.

Reply:
82d6ed9cf9d732b16b348287393b2781030b3a30 should extend the capture of the git hash and count information. 

I believe that the check for `"$versionsuffix" != "$gitversion"` should ensure that the tag name capture is OK, too.